### PR TITLE
Rubocop

### PR DESCRIPTION
**Pull request by innatewonderer**
Adding the `cop` to help the app be more stylish

### CODE REVIEW

File: app/concerns/listens.rb [hunk 1]
```diff
@@ -95,24 +97,24 @@ def play_local_mp3
...
-    if !request.headers["Range"] or request.headers["Range"]=="bytes=0-"# browser wants the whole file
+    if !request.headers["Range"] || (request.headers["Range"] == "bytes=0-") # browser wants the whole file
       status = "200 OK"
       headers["Content-Length"] = (file_end.to_i - file_begin.to_i + 1).to_s
 
-      send_file file_to_send, :type => 'audio/mpeg', :disposition => 'attachment;',
-      :url_based_filename => true, :status => status, :stream => true, :buffer_size  =>  4096
+      send_file file_to_send, type: 'audio/mpeg', disposition: 'attachment;',
+                              url_based_filename: true, status: status, stream: true, buffer_size: 4096
     else
```
Comment (sudara) [inline app/concerns/listens.rb]: Personally not a fan of this style of indentation, is this something we can disable for now or set to two spaces? (vs. aligned with the first argument)

File: app/controllers/assets_controller.rb [hunk 2]
```diff
@@ -34,26 +34,27 @@ def latest
...
-    else
-      @assets = @user.assets.published
-    end
-    @assets = @assets.recent.paginate(:per_page => 200, :page => params[:page])
+    @assets = if current_user_is_admin_or_owner?(@user)
+                @user.assets
+              else
+                @user.assets.published
+              end
+    @assets = @assets.recent.paginate(per_page: 200, page: params[:page])
```
Comment (sudara) [inline app/controllers/assets_controller.rb]: Same with this kind of indentation, kinda bums me out. The code here wasn't great, but I'd prefer it not to rewrite if/else blocks.

File: app/concerns/listens.rb [hunk 3]
```diff
@@ -52,19 +52,21 @@ def prevent_abuse(asset)
...
+        ip: request.remote_ip,
+        city: request.headers["HTTP_GEOIP_CITY"], # set by nginx geoip
+        country: request.headers["HTTP_GEOIP_COUNTRY_CODE"]
+      )
+    end
   end
 
   def ip_just_registered_this_listen?(asset)
-    last_listen = asset.listens.since(1.week.ago).where(:ip => request.remote_ip).first
+    last_listen = asset.listens.since(1.week.ago).where(ip: request.remote_ip).first
```
Comment (sudara) [inline app/concerns/listens.rb]: So happy to finally remove the hashrockets...

File: app/models/playlist.rb [hunk 4]
```diff
@@ -26,68 +26,68 @@
...
-  scope :albums,           -> { where(:is_mix => false).where(:is_favorite => false)                              }
-  scope :favorites,        -> { where(:is_favorite => true)                                                       }
-  scope :only_public,      -> { where(:private => false).where(:is_favorite => false).where("tracks_count > 1")   }
-  scope :include_private,  -> { where(:is_favorite => false)                                                      }
+  scope :mixes,            -> { where(is_mix: true) }
+  scope :albums,           -> { where(is_mix: false).where(is_favorite: false) }
+  scope :favorites,        -> { where(is_favorite: true) }
+  scope :only_public,      -> { where(private: false).where(is_favorite: false).where("tracks_count > 1") }
+  scope :include_private,  -> { where(is_favorite: false) }
   scope :recent,           -> { order('playlists.created_at DESC')                                                }
```
Comment (sudara) [inline app/models/playlist.rb]: Yay!

File: app/models/playlist.rb [hunk 5]
```diff
@@ -26,68 +26,68 @@
...
-  has_one  :pic, :as => :picable, :dependent => :destroy
+  belongs_to :user, counter_cache: true
+  has_one  :pic, as: :picable, dependent: :destroy
   has_many :tracks,
-     -> {order(:position).includes(:asset => :user)},
-     :dependent => :destroy
-  has_many :assets, :through => :tracks
+           -> { order(:position).includes(asset: :user) },
+           dependent: :destroy
+  has_many :assets, through: :tracks
```
Comment (sudara) [inline app/models/playlist.rb]: boo! hehe

**Comment by innatewonderer** [comment]
I'll create a new PR since we want the spacing issue to be ignored by rubocop.

**Pull request closed without merging**